Src/library/base/elks/kernel/type.e

```diff
@@ -1,12 +1,13 @@
 note
 	description: "Representation of an Eiffel type."
+	remark: "At any given time, there is no more than one instance of TYPE representing a given Eiffel type."
```

Comment:
Document the fact that {ANY} and {ANY} are the same object (i.e. {ANY} = {ANY}).